app/helpers/issues_helper.rb

```diff
@@ -26,9 +26,9 @@ def reportable_title(reportable)
   end
 
   def open_issues_count
-    count = Issue.visible_to(current_user).open.limit(100).size
-    if count > 99
-      tag.span("99+", :class => "count-number")
+    count = Issue.visible_to(current_user).open.limit(Settings.max_issues_count).size
```

Comment:
So I think this will all be more straightforward if the setting is the same as the digits shown. So if you want to show `99+` you set the limit to `99`. It's strange to set the limit to "100" in then a different number to show up.

Additionally, 99+ just means "at least" 99, so it's a valid display for both 99 and 100. The original implementation checks for `99+1` and then shows either `99` or `99+`, but this is unnecessary complexity. Better just to go `97`, `98`, `99+`, and this means no +1 maths is necessary.